store/etcd/rulesets.go

```diff
@@ -519,6 +512,6 @@ func (s *RulesetService) signaturesPath(p string) string {
 	return path.Join(s.Namespace, "rulesets", "signatures", p)
 }
 
-func (s *RulesetService) latestRulesetPath(p string) string {
+func (s *RulesetService) latestRuleset(p string) string {
```

Comment:
You should not rename this function, this is still returning the etcd path for the latest ruleset

Reply:
Did you see that commit https://github.com/heetch/regula/pull/57/commits/85a65c96aa2516661932263c17a8d3f3fce7dc64 ?

Reply:
I've just seen it but I don't understand